src/sdk/pynni/nni/compression/torch/compressor.py

```diff
@@ -200,6 +202,25 @@ def _wrap_modules(self, layer, config):
         raise NotImplementedError()
 
 
+    def add_activation_collector(self, collector):
```

Comment:
could you update the two activation pruning algorithm to show how to use this collector?

Reply:
updated.